nextflow/modules/annotation/ParseAlphaMissenseIntoHt/main.nf

```diff
@@ -6,18 +6,16 @@ process ParseAlphaMissenseIntoHt {
     publishDir params.processed_annotations, mode: 'copy'
 
     input:
-        path(am_tsv)
+        path am_tsv
 
     output:
-        path("alphamissense_38.ht.tar")
+        path "alphamissense_38.ht"
 
     script:
         """
         ParseAlphaMissenseIntoHt \
             --am_tsv ${am_tsv} \
             --ht_out alphamissense_38.ht
-        tar --no-xattrs -cf alphamissense_38.ht.tar alphamissense_38.ht
-        rm -r alphamissense_38.ht
```

Comment:
This requires an entry in the changelog - we now expect this data to remain un-tar'd. This saves substantially on pre-processing time in this stage